modules/cf-core/src/api.ts

```diff
@@ -35,11 +37,13 @@ const controllers = [
   CreateChannelController,
   DepositController,
   InstallAppInstanceController,
+  InstallBalanceRefundController,
```

Comment:
A better name might be "RequestDepositRightsController" or something more descriptive of what the intent is.

Reply:
Thanks, this is a great name.